src/app/products/components/product-delete-confirm-dialog/product-delete-confirm-dialog.component.ts

```diff
@@ -0,0 +1,23 @@
+import { Component, Inject, OnInit } from '@angular/core';
+import { MatDialogRef, MAT_DIALOG_DATA } from '@angular/material/dialog';
+import { Product } from 'src/app/shared/models/product';
+
+@Component({
+  selector: 'tk-product-delete-confirm-dialog',
+  templateUrl: './product-delete-confirm-dialog.component.html',
+  styleUrls: ['./product-delete-confirm-dialog.component.scss']
+})
+export class ProductDeleteConfirmDialogComponent implements OnInit {
+
+  constructor(
+    public dialogRef: MatDialogRef<ProductDeleteConfirmDialogComponent>,
```

Comment:
you do not need to mark it explicitly as public (it's marked as that by default). also since you are not reassigning it mark it as readonly

Reply:
They had to be marked as public in constructor (otherwise app would not compile), but I marked them as readonly as advised:)